src/bootstrap.jl

```diff
@@ -1,6 +1,56 @@
 # A collection of bootstrapped code which should be generated
 # dynamically in future.
 
+struct RecoveredTBasket
+    data
```

Comment:
Nono, I also don't do anything with the baskets yet. Currently it's just a "proper" readout so that everything is parsed (more or less).

I'm working on the next PR ;)